### ISSUE TITLE
scale differ from ios to android

### ISSUE THREAD

**Issue by miias** [CariusLars/ar_flutter_plugin#93]
hello everyone
so i have some 3d objects that im spawning on a plane on android they have a normal scale but on ios their scale is not the same eventho i set the starting scale to (1,1,1) like so
`          object = ARNode(
              type: NodeType.fileSystemAppFolderGLB,
              uri: 'model.glb',
              scale: Vector3(prevScale, prevScale, prevScale));
`
prevScale = 1
![photo_2022-03-01_12-17-48](https://user-images.githubusercontent.com/61624285/156150837-f2f0393a-cc6b-4750-bc3f-ee1cda90a379.jpg)

here's the model on android

![WhatsApp Image 2022-02-28 at 12 13 05 PM](https://user-images.githubusercontent.com/61624285/156150980-ad691f35-81ba-41d1-841e-b01e65511c76.jpeg)

and here's the model on ios from about the same distance

**Comment by resie001** [comment]
yeah, dunno why that's my model have different scale in iOS.

**Comment by leeprobert** [comment]
The internal code scales down the Android models to 0.01 so you need to upscale it back up in your flutter code. I am still unclear why this was done. @CariusLars - can you let us know what the logic was for this originally?

**Comment by resie001** [comment]
Hey @miias , recently i fork this repo and make little adjustment for iOS. If you wanna try it, you can use this [repo](https://github.com/resie001/ar_flutter_plugin). but don't make high hope for this repo though 😅

**Comment by ehabme** [comment]
any update guys ? i have the same problem ?
i don't have a real phone so i upload alot to testfight for test the same issue :(
what can i do ? 
i tried to set scale as (1.0,1.0,1.0) but no new !

**Comment by resie001** [comment]
hey @miias , maybe use my [repo](https://github.com/resie001/ar_flutter_plugin), forked from this repo. But don't make high expectation for this forked repo though, i just college student 😅.

**Comment by resie001** [comment]
hey @ehabme , maybe use my [repo](https://github.com/resie001/ar_flutter_plugin), forked from this repo. But don't make high expectation for this forked repo though, i just college student 😅.

**Comment by ehabme** [comment]
> hey @ehabme , maybe use my [repo](https://github.com/resie001/ar_flutter_plugin), forked from this repo. But don't make high expectation for this forked repo though, i just college student 😅. I just edited for iOS code that have scale as 1.0, for my case it just worked.

yes , i got it bro , i did the same you said and working good , thank you bro , and all plugin developers

**Comment by remakerich** [comment]
Guys,
Sorry I'm a bit off topic, but
Is it normal that on iOS objects are rendered without any lighting... they look like raw 3D models, very bad to look at, cuz they have no fidelity at all.

On Android objects look decent, because theres some lighting fx and even some shadows...

Does someone know how to fix it?
